sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -83,7 +83,7 @@ class SQLAppStatusListener(
     // track of the metrics knows which accumulators to look at.
     val accumIds = exec.metrics.map(_.accumulatorId).sorted.toList
```

Comment:
If we are going to make Set, I guess we don't need to sort it, and may also don't need to convert to List and Set again.

Does changing List to Set affect any content on UI page? Just wanted to double check why we have been sorting the accumulator ids, and whether this patch breaks the intention or not.

Reply:
Good point. If we use this as `set`, `sort` is not necessary.
When I have just looked at the code, `accumulatorIds` does not take care of the order of elements. I would appreciate it if someone double-checks it.

Reply:
Thanks @HeartSaVioR @kiszk